modules/administration-guide/partials/assembly_backing-up-of-che-instances-to-a-rest-backup-server.adoc

```diff
@@ -0,0 +1,12 @@
+[id="backing-up-of-{prod-id-short}-instances-to-a-rest-backup-server_{context}"]
+= Backing up of {prod-short} instances to a REST backup server
+
+You can manage backups of {prod-short} instances to a REST backup server with custom resources or {prod-cli}:
```

Comment:
It is not exactly manage, it is more send

Reply:
@mmorhun Thank you. I've also fixed the same in the SFTP, Amazon S3, and the internal backup server files.